### PR TITLE
Instructor Dashboard

### LINKED ISSUE #29: Instructor Dashboard
Should have an instructor dashboard as per mockups:

https://github.com/PestoTech/project-class-insights-client/blob/master/mockups/Instructor-Dashboard.png

### PR DESCRIPTION
**Pull request by kundanvishen**
This PR will resolve #29

### CODE REVIEW

**Comment by kundanvishen** [comment]
class_summary_card

**Comment by kundanvishen** [comment]
drop_down_instructor_dashboard_desktop_lowframes
drop_down_instructor_dashboard_iphone_x

File: src/components/Charts/dashboard.chart.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+      </Group>
+    </svg>
+
+  );
+};
+
+
+class DashboardChart extends React.Component {
+  render() {
+    return (<BarGraph />);
```
Comment (anshurathee) [inline src/components/Charts/dashboard.chart.js]: Why parenthesis ?

Comment (anshurathee) [reply src/components/Charts/dashboard.chart.js]: @kundanvishen

Comment (kundanvishen) [reply src/components/Charts/dashboard.chart.js]: @anshurathee Dashboard chart would contain other information apart from the BarGraph component.
But removing it for now. 👍🏻

**Review by anshurathee — APPROVED** [review]

**Pull request merged**